install.sh

```diff
@@ -103,24 +103,29 @@ check_download() {
 
     # Checkout the correct stream
     cd $1
-    git checkout origin/${STREAM}
```

Comment:
Why this? I want to check out the upstream branch, not local one.

Reply:
Doing this doesn't checkout the upstream branch, it leaves the repo in a detached head state.

Reply:
It checkouts the upstream branch, which is exactly what a detached head state is.

Reply:
Well not exactly, but `origin/${STREAM}` is what I meant and I think it should stay this way.

Reply:
Is there a way you can skip this patch or would I have to take the patch out of the series?

Reply:
You can just rebase and drop the patch.